api/db/migrations/20190820092603_add_several_attributes_to_student.js

```diff
@@ -0,0 +1,33 @@
+const TABLE_NAME = 'students';
+
+exports.up = function(knex) {
+  return knex.schema.table(TABLE_NAME, (table) => {
+    table.string('preferredName');
+    table.string('middleName');
+    table.string('thirdName');
+    table.string('birthCity');
+    table.string('birthProvince');
+    table.string('birthCountry');
+    table.string('basicTrainingModule');
```

Comment:
J'ai du mal à comprendre de quoi il s'agit.
Plus généralement, je remets en cause l'utilisation systématique de l'anglais. 

Ici on fait référence au ``CODE_MEF``de l'étudiant, qui précise ses éventuelles options (si j'ai bien compris)
En renommant la colonne ``mefCode`` directement, on se rapprocherait du vocabulaire métier.

Reply:
je suis pas forcément pour, par exemple pour les CGUs, on est partis sur Terms of Service, même en base. Donc histoire d'être cohérent, je laisserais en anglais. Par contre, si une autre formulation anglaise est mieux, je suis preneuse :)

Reply:
Je suis assez d'accord avec thierry. Le problème de `basicTrainingModule` c'est qu'en le lisant on sait pas ce que c'est. Ça me semblerait une bonne idée si on s'attend à généraliser le concept à d'autres pays, et qu'on chercherait un terme qui regrouperait différentes notions. Mais là j'ai l'impression que c'est franco-français. Sinon pour `termsOfServices` pour le coup c'était non-ambigu, c'est une traduction 1-1. Je lancerai bien une discussion slack pour élargir, j'ai un avis mais pas complètement tranché.

Reply:
Je suis d'accord avec @octo-thlo et @Radioreve, `basicTrainingModule` me paraît abusif. Il arrive que des données représentent une structure locale qui n'a pas vocation à être traduite. Autre exemple, "chef-lieu de canton"… Dans ces cas-là il vaut mieux utiliser le mot du métier dans la langue du métier plutôt que d'inventer un mot qui n'existe pas en anglais.

Reply:
changé pour `MEFCode`